tox.ini

```diff
@@ -34,6 +34,9 @@ extras =
     windows
 commands_pre =
     find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -path '*/__pycache__/*' -name '*.py[c|o]' -delete
+    # temporary ignoring the exit code of pip check until we address azure
+    # extra conflicts: https://github.com/ansible/ansible/issues/53761
+    - pip check
```

Comment:
I think linting doesn't belong to the testing section. It's a separate thing. It's closer to metadata validation.

Reply:
`pip check` is not linting, is detecting conficts between python packages. We cannot do it in linting because it is possible to have conflicts on each specific environment - really possible to discover that we have a conflict only on python27 with ansible26. 

Obviously that we cannot afford to add extra environments only for this purpose.'

Also this test is very quick, like less than 1s.

Reply:
Okay. Agreed.